enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1543,14 +1562,18 @@ func (c *FirecrackerContainer) getConfig(ctx context.Context, rootFS, containerF
 		netnsPath = c.network.NamespacePath()
 	}
 	bootArgs := getBootArgs(c.vmConfig)
+	guestKernelPath := c.executorConfig.GuestKernelImagePath
+	if c.shouldUpgradeGuestKernel() {
```

Comment:
is there a reason for not putting this (and the version tweak above) where `GuestKernelImagePath` is passed in?

Reply:
oops oops, got it